pkg/sync/seqatomic/seqatomic_test.go

```diff
@@ -118,14 +118,15 @@ func BenchmarkSeqAtomicTryLoadIntUncontended(b *testing.B) {
 }
 
 // For comparison:
-func BenchmarkAtomicValueLoadIntUncontended(b *testing.B) {
-	var a atomic.Value
```

Comment:
Ditto, we should not change this as well, since variable `a` is intending to store `int` value, not pointer.